src/h5web/visualizations/shared/TooltipMesh.tsx

```diff
@@ -1,10 +1,17 @@
-import React, { ReactElement, useCallback } from 'react';
+import React, { ReactElement, useCallback, useContext } from 'react';
 import { Dom, PointerEvent, useThree } from 'react-three-fiber';
 import { TooltipWithBounds, useTooltip } from '@vx/tooltip';
 import { Line } from '@vx/shape';
 
 import styles from './TooltipMesh.module.css';
-import { useAbscissaScale, useOrdinateScale } from './utils';
+import { getAxisScale } from './utils';
+import AxisSystemContext from './AxisSystemContext';
+
+const GUIDE_PROPS = {
+  stroke: 'gray',
+  strokeWidth: 1.5,
+  strokeOpacity: 0.5,
+};
```

Comment:
I just moved this static object out of the component and renamed it for consistency.